api/lib/schema.js

```diff
@@ -110,6 +110,7 @@ const typeDefs = gql`
     beneficiary: NetworkAccount
     summary: String
     parameter: String
+    changes: Changes
```

Comment:
```suggestion
    changes: String
```
This is the right way

Reply:
I think we should instead actually have Changes and map in the FE. this way we can actually map them nice line in a table or sth. get rid of parameter as well.